stdlib/public/SDK/Foundation/NSCoder.swift

```diff
@@ -229,16 +229,3 @@ extension NSKeyedUnarchiver {
     }
   }
 }
-
-
-@available(*, deprecated, renamed:"NSCoding", message: "Please use NSCoding")
-typealias Coding = NSCoding
```

Comment:
These aren't public, so we can just delete them. Oops.
(Confirmed that attempting to use these gives `use of undeclared identifier`)